IFC4x3/Properties/s/SignalHoodLength_0V5henCzj7ouBew9_bcs0i/Documentation.md

```diff
@@ -0,0 +1 @@
+Type of the cover of the signal lamp against glaring sun.
```

Comment:
This really does not match with the property name and type.

Reply:
Updated documentation: https://github.com/bSI-InfraRoom/IFC-Specification/pull/179/commits/b9fc777a565534c2a373dc537fcde287347974bb